rclpy/guard_conditions/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>examples_rclpy_guard_conditions</name>
+  <version>0.0.1</version>
```

Comment:
This version should match the same as the others in this repository so our tooling works smoothly (e.g. `catkin_prepare_release` and `bloom`).
```suggestion
  <version>0.9.2</version>
```